sphinx/main_gallery.json

```diff
@@ -20,8 +20,6 @@
         { "path": "examples/plotting/file/lorenz.py",          "name": "lorenz"                                      },
         { "path": "examples/plotting/file/color_scatter.py",   "name": "color_scatter"                               },
         { "path": "examples/plotting/file/color_sliders.py",   "name": "color_sliders"                               },
-        { "path": "examples/models/iris_splom.py",             "name": "iris_splom",             "symbol": "grid"    },
-        { "path": "examples/models/anscombe.py",               "name": "anscombe",               "symbol": "grid"    },
```

Comment:
These are the two examples I had to remove because they do explicit `Document` management. It's possibble `Document` could be patched to a function that returns `curdoc()` tho